e2e/tests/tutorial.spec.ts

```
@@ -0,0 +1,52 @@
+        await tutorial.dismissTutorial();
+        await tutorial.refreshPage();
+        expect(await tutorial.isContentInCenterHidden()).toBeTruthy();
+        await tutorial.changeLocalStorage('false');
+    })
+
+    test("@admin validate that clicking away dismisses the tutorial panel", async () => {
+        const tutorial = await browser.createNewPage(TutotialPanel, urls.graphUrl);
+        await tutorial.changeLocalStorage('true');
+        await tutorial.refreshPage();
+        await tutorial.clickAtTopLeftCorner();
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

ESLint

[error] Return values from promise executor functions cannot be read.

(no-promise-executor-return)